ext/rbs_extension/ruby_objs.h

```diff
@@ -43,9 +45,28 @@ VALUE rbs_tuple(VALUE types, VALUE location);
 VALUE rbs_type_name(VALUE namespace, VALUE name);
 VALUE rbs_union(VALUE types, VALUE location);
 VALUE rbs_variable(VALUE name, VALUE location);
-
 VALUE rbs_ast_directives_use(VALUE clauses, VALUE location);
 VALUE rbs_ast_directives_use_single_clause(VALUE type_name, VALUE new_name, VALUE location);
 VALUE rbs_ast_directives_use_wildcard_clause(VALUE namespace, VALUE location);
+VALUE rbs_ast_members_attr_accessor(VALUE name, VALUE type, VALUE ivar_name, VALUE kind, VALUE annotations, VALUE location, VALUE comment, VALUE visibility);
+VALUE rbs_ast_members_attr_reader(VALUE name, VALUE type, VALUE ivar_name, VALUE kind, VALUE annotations, VALUE location, VALUE comment, VALUE visibility);
+VALUE rbs_ast_members_attr_writer(VALUE name, VALUE type, VALUE ivar_name, VALUE kind, VALUE annotations, VALUE location, VALUE comment, VALUE visibility);
+VALUE rbs_ast_members_class_instance_variable(VALUE name, VALUE type, VALUE location, VALUE comment);
+VALUE rbs_ast_members_class_variable(VALUE name, VALUE type, VALUE location, VALUE comment);
+VALUE rbs_ast_members_extend(VALUE name, VALUE args, VALUE annotations, VALUE location, VALUE comment);
+VALUE rbs_bases_void(VALUE location);
+VALUE rbs_bases_top(VALUE location);
+VALUE rbs_bases_self(VALUE location);
+VALUE rbs_bases_nil(VALUE location);
+VALUE rbs_bases_instance(VALUE location);
+VALUE rbs_bases_class(VALUE location);
+VALUE rbs_bases_bottom(VALUE location);
+VALUE rbs_bases_bool(VALUE location);
+VALUE rbs_bases_any(VALUE location);
```

Comment:
All 9 of these types used to have their own Ruby classes, but shared the same "constructor" (`rbs_base_type()`).

We will eventually want 9 different C structs to model these (to keep a simple 1:1 correspondence to the Ruby classes), with 9 different constructor functions. We did that here, since it simplifies automation.